sky/engine/core/script/dom_dart_state.h

```diff
@@ -35,6 +36,9 @@ class DOMDartState : public DartState {
   Dart_Handle value_handle() { return value_handle_.value(); }
   Dart_Handle color_class() { return color_class_.value(); }
 
+  void set_font_selector(PassRefPtr<FontSelector> selector);
+  PassRefPtr<FontSelector> font_selector();
```

Comment:
Usually we'd return `FontSelector*` because we're not actually able to donate a refcount here so there's no advantage to using PassRefPtr.